src/main/java/io/github/railroad/project/ui/create/widget/ProjectSortComboBox.java

```diff
@@ -1,42 +1,41 @@
 package io.github.railroad.project.ui.create.widget;
 
 import io.github.railroad.project.ProjectSort;
+import io.github.railroad.ui.localized.LocalizedComboBox;
+import io.github.railroad.utility.localization.L18n;
 import javafx.collections.FXCollections;
-import javafx.scene.control.ComboBox;
 import javafx.scene.control.ListCell;
-import javafx.util.StringConverter;
 import org.kordamp.ikonli.fontawesome5.FontAwesomeSolid;
 import org.kordamp.ikonli.javafx.FontIcon;
 
-import java.util.Locale;
-
-public class ProjectSortComboBox extends ComboBox<ProjectSort> {
+public class ProjectSortComboBox extends LocalizedComboBox<ProjectSort> {
     private static final FontIcon ICON = FontIcon.of(FontAwesomeSolid.FILTER);
 
     public ProjectSortComboBox() {
+        super(ProjectSort::getName, ProjectSort::valueOf);
         setItems(FXCollections.observableArrayList(ProjectSort.values()));
         setPrefWidth(120);
-        setConverter(new StringConverter<>() {
-            @Override
-            public String toString(ProjectSort object) {
-                return object.getName();
-            }
-
-            @Override
-            public ProjectSort fromString(String string) {
-                return ProjectSort.valueOf(string.toUpperCase(Locale.ROOT));
-            }
-        });
 
         setButtonCell(new ListCell<>() {
             @Override
             protected void updateItem(ProjectSort item, boolean empty) {
                 super.updateItem(item, empty);
-                setText(item == null ? null : item.getName());
+                setText(item == null ? null : L18n.localize(item.getName()));
+                L18n.currentLanguageProperty().addListener((observable, oldValue, newValue) -> {
+                    if (item != null) {
+                        setText(L18n.localize(item.getName()));
+                    } else {
+                        setText(null);
+                    }
+                });
                 setGraphic(ICON);
             }
         });
 
+        L18n.currentLanguageProperty().addListener((observable, oldValue, newValue) -> {
+            setItems(FXCollections.observableArrayList(ProjectSort.values()));
+        });
+
         setValue(ProjectSort.NONE);
     }
```

Comment:
Any reason you can't use a `LocalizedListCell`?

Reply:
Because that isn't a dropdown? Unless I'm missing something. Changing it to a localized list cell just removes the dropdown part of it and messes with theming.